src/partition.mjs

```diff
@@ -0,0 +1,42 @@
+import Dequeue from 'dequeue'
+import iterable from './iterable'
+
+const UNDONE = value => ({ done: false, value })
+const DONE = { done: true, value: undefined }
```

Comment:
a matter of style. I found more readable if you just remove these constants and be more explicit

Reply:
But then [these lines](https://github.com/sithmel/iter-tools/blob/ea733dd5cf86d508cdedeb6fd44b9e4047271064/src/partition.mjs#L26-L27) would be ugly.

Reply:
They would be explicit and obvious :smile:

Reply:
I guess it doesn't matter since I'm going to replace the whole `part()` function with a for-loop anyway.